### ISSUE TITLE
Disable export PDF button

### ISSUE THREAD

**Issue by mchenachi** [mbrn/material-table#2216]
How can I disable or remove the export PDF button option?? I didn't found it on the docs

![Capture](https://user-images.githubusercontent.com/32645315/88093352-456c8a80-cb68-11ea-983f-6fa40b899360.PNG)

**Comment by janderson215** [comment]
I couldn't find anything to remove this either. This is a very temporary workaround because the users will wonder why they can't export to PDF and I think `alert`s are a no-no in React, but this is what I'm doing. In the `options` prop of `MaterialTable`, add 
`exportPdf: () => alert('This feature is disabled. Data can only be exported as a CSV.')`

**Comment by AlexPetit12** [comment]
I am also looking for a solution to this issue on how to hide/remove "Export as PDF".

**Comment by bturtu405** [comment]
I'm also looking for solution

**Comment by priyanshu-intellemo** [comment]
+1

**Comment by b3ni** [comment]
+1

**Comment by erikasoares** [comment]
+1

**Comment by phanvantien270290** [comment]
> I am also looking for a solution to this issue on how to hide/remove "Export as PDF".

and i think catch event for them ( like exportCSV event)

**Comment by omnislashbr** [comment]
If you are desperate, as a temporary solution, you can select it with css, then change its display, here is the selector:

li.MuiButtonBase-root.MuiListItem-root.MuiMenuItem-root.MuiMenuItem-gutters.MuiListItem-gutters.MuiListItem-button:nth-child(2) {
display:none !important;
}

**Comment by cassidylaidlaw** [comment]
This is a slightly more robust workaround in plain JavaScript, using a MutationObserver to determine when the export menu opens and then removing the Export as PDF menu item:

    setTimeout(() => { // Wait until page load.
      new MutationObserver((mutations, observer) => {
        mutations.forEach((mutation) => {
          mutation.addedNodes.forEach((addedNode) => {
            if (addedNode.nodeType === Node.ELEMENT_NODE) {
              const element = addedNode;
              if (element.classList.contains('MuiPopover-root')) {
                Array.from(
                  element.getElementsByClassName('MuiMenuItem-root'),
                ).forEach((menuItem) => {
                  if (menuItem.textContent === 'Export as PDF') {
                    menuItem.remove();
                  }
                });
              }
            }
          });
        });
      }).observe(document.body, {childList: true});
    }, 0);

**Comment by dj0nes** [comment]
This is fixed as of 1.6.9 via https://github.com/mbrn/material-table/pull/2335

So now you can opt in to only the csv button:

```jsx
<MaterialTable options={{ exportButton: { csv: true }} />
```

**Comment by zeckdude** [comment]
This issue can be closed

**Comment by janderson215** [comment]
Using 1.69.2 and the correct syntax to exclude the PDF option is:
```js-jsx
<MaterialTable options={{ exportButton: { csv: true, pdf: false }} />
```

**Comment by servonlewis** [comment]
> Using 1.69.2 and the correct syntax to exclude the PDF option is:
> 
> ```js-jsx
> 
> <MaterialTable options={{ exportButton: { csv: true, pdf: false }} />
> 
> ```
> 
> 

Would love to use it but did they fix that massive memory issue in 1.69? I had to go back to 1.68 to be able to have data that updates in this table.
